class.jetpack-connection-banner.php

```diff
@@ -150,7 +150,9 @@ public static function enqueue_connect_button_scripts() {
 
 		$jetpackApiUrl = parse_url( Jetpack::connection()->api_url( '' ) );
 
-		if ( Constants::is_true( 'JETPACK_SHOULD_USE_CONNECTION_IFRAME' ) ) {
+		if ( wp_is_mobile() ) {
```

Comment:
Instead of relying on `wp_is_mobile()`, what do you think about using our own `jetpack_is_mobile()`, which is a bit more detailed?

Reply:
https://github.com/Automattic/jetpack/pull/13364/commits/fac87766911811ccf5b3688c354f51b557124e39 good idea